index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header> 
+         <p>Jhon Neira</p>
+         <div class="menu-icon">
+            <img src="Images/Menu.png" alt="menu-icon">
+         </div> 
+    </header>
+    <main>
+        <h1>
+            Hey There. <br>
+            I'm Alejandro
+        </h1>
+        <h2>
+            I am a Software Developer
+        </h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don't hestiate to contact me.
+        </p>
+        <div class="Social-container">
+            <img class="Gato" src="Images/Gato.png" alt="GitHub">
+            <img src="Images/In.png" alt="LinkedIn">
```

Comment:
- [ ] Please add a class for all your icons and give them a standard width and height in your styles file to follow the Figma design

As you can see in your design, the first icon is ok, but the rest icons don't have a good size.
![image](https://user-images.githubusercontent.com/82678907/148315881-2a3d24f1-e573-46c0-ad7f-7c32ea545612.png)

Reply:
@Felipeg005 I did as you say but the images still are in different sizes